test/article/model.js

```diff
@@ -1,58 +0,0 @@
-/**
- * Module dependencies.
- */
-var should = require('should'),
```

Comment:
Any reason why you deleted this file?

Reply:
We removed it when Idrees and I were testing. I will restore it now.